src/data-table/__tests__/column-categorical.test.js

```diff
@@ -14,6 +14,31 @@ import {CategoricalColumn} from '../index.js';
 
 let container: HTMLDivElement;
 
+// at this point it's likely worth pulling in react-testing-library, but
+// it seems to require the latest React@16.9 with async act. this maybe a
```

Comment:
We can use 16.9 in Base and still allow 16.8 for consumers? There are not really any new features (except testing and profiling) that we could use in components, so it should be safe.

Reply:
Cool, I agree that should be fine. I can put up another pr that removes this util and adds `react-testing-library`